src/Core/defaultValue.js

```diff
@@ -7,7 +7,7 @@
 
 import { Vector3 } from 'three';
 
-var defaultValue = function defaultValue(value, def) {
+const defaultValue = function defaultValue(value, def) {
```

Comment:
What about removing this `defaultValue` thing completely?
Most of the time `a = defaultValue(foo, bar)` can be replaced with `a = foo || bar`